unlock-app/src/services/storageService.ts

```diff
@@ -71,16 +71,20 @@ export class StorageService extends EventEmitter {
   }
 
   async signOut() {
-    const endpoint = `${this.host}/v2/auth/revoke`
-    const response = await fetch(endpoint, {
-      method: 'POST',
-      headers: {
-        'refresh-token': this.refreshToken!,
-      },
-    })
-    if (response.ok) {
+    try {
+      const endpoint = `${this.host}/v2/auth/revoke`
+      await fetch(endpoint, {
+        method: 'POST',
+        headers: {
+          'refresh-token': this.refreshToken!,
+        },
+      })
       localStorage.removeItem(`locksmith-access-token`)
```

Comment:
Actually, can you also include that in the the try statement? In incognito,accessing localStorage throws.